src/components/Front/Front.css

```diff
@@ -1,53 +1,29 @@
-
-
-.welcome{
-  /* min-height: 100vh; */
+.content {
+  min-height: 50vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
 }
 
-
-.introduction{
-	font-size: 5rem;
-}
-#one{
-margin-right: 46rem;
-}
-#two{
-margin-right: 15rem;
+.welcome {
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
-.content{
-	 text-align:center;
-	margin-top: 10rem; 
-  
-	
-}
-.introduction{
-  text-transform:uppercase;
-	font-size:5rem;
-	font-family:'Verdana';
-	/* padding:30px; */
+
+.introduction {
+  text-transform: uppercase;
```

Comment:
Major problem: Please maintain some limited different font-families, otherwise the project started looking like a junk yard.

Reply:
ok, noted